frontend/pages/hosts/details/DeviceUserPage/ManualEnrollMdmModal/ManualEnrollMdmModal.tsx

```diff
@@ -104,7 +104,7 @@ const ManualEnrollMdmModal = ({
           </li>
           <li>
             In the search bar, type “Profiles”. Select <b>Profiles</b>, double
-            click <b>Enrollment Profile</b>, and select <b>Install</b>.
+            click <b>Fleet Enrollment</b>, and select <b>Enroll</b>.
```

Comment:
```suggestion
            click <b><your organization's name> Enrollment</b>, and select <b>Enroll</b>.
```

Hey! This profile name is actually the org name in Fleet + "Enrollment." So for us, the profile would be called "Fleet Enrollment" for ACME Inc the profile would be called "ACME Inc Enrollment"

Reply:
Nice catch, Noah! 🦅 👁️

Reply:
Got it, will change that